Resources/public/js/views/fields/ez-image-editview.js

```diff
@@ -51,6 +51,55 @@ YUI.add('ez-image-editview', function (Y) {
         },
 
         /**
+         * Set the file attribute with a struct based on the file object from the
+         * input file element. This implementation adds a validation to check that the chosen file
+         * is really an image the browser can load.
+         *
+         * @method _base64ToFileStruct
+         * @protected
+         * @param {EventFacade} e event facade
+         * @param {File} file the File object from the input file element
+         */
+        _base64ToFileStruct: function (file, e) {
+            var base64 = e.target.result.replace(/^.*;base64,/, ''),
+                testImageObject = new Image(100, 200),
+                fileStruct = this._createFileStruct(file, base64);
+
+            testImageObject.src = fileStruct.displayUri;
```

Comment:
you should probably move that line after setting the error and load event handlers.